eywa/nlu/entity_extractor_utils.py

```diff
@@ -0,0 +1,18 @@
+"""Checks, utilities for entity_extractor
```

Comment:
* This method doesnt seem to be specific for entity extractor.
* No utils allowed in eywa/nlu. Utils should go to eywa/utils.
* f(X, Y) in entity extractor should possibly accept single samples as well.

Reply:
Made the specified changes.